packages/typedoc-plugin-appium/lib/model/types.ts

```diff
@@ -0,0 +1,96 @@
+import {Comment, DeclarationReflection, ProjectReflection} from 'typedoc';
+import {CommandInfo} from './command-info';
+
+/**
+ * WD spec allows these HTTP methods.
+ */
+export type AllowedHttpMethod = 'GET' | 'POST' | 'DELETE';
+/**
+ * An Express-style "route", otherwise known as a "URI Template"
+ *
+ * Aliased for intent
+ */
+export type Route = string;
+
+/**
+ * A method name in `BaseDriver` or subclass thereof
+ *
+ * Currently must be a string, but could theoretically be a {@link PropertyKey}
+ *
+ * Aliased for intent
+ */
+export type Command = string;
```

Comment:
does TS support the concept similar to python's [NewType](https://justincaustin.com/blog/python-typing-newtype/) or this is just a transparent type alias?

Reply:
It's just an alias to know that we're talking about a string representing a command name.  Likewise with `Route`